src/ontology/reports/doid.subclass.confirmed.robot.tsv

```diff
@@ -4537,6 +4532,7 @@ MONDO:0008558	autoimmune thrombocytopenic purpura	MONDO:0004680	DOID:8924	DOID:8
 MONDO:0008585	HELLP syndrome	MONDO:0001641	DOID:13133	DOID:13129	severe pre-eclampsia
 MONDO:0008590	tremor, hereditary essential, 1	MONDO:0003233	DOID:0111428	DOID:4990	essential tremor
 MONDO:0008596	trichorhinophalangeal syndrome type I	MONDO:0000426	DOID:14743	DOID:0050736	autosomal dominant disease
+MONDO:0008597	obsolete trichorhinophalangeal syndrome, type III	MONDO:0000426	DOID:0080376	DOID:0050736	autosomal dominant disease
```

Comment:
Not obsolete Mondo class in confirmations - that is pretty dangerous for the downstream processing as it will add a subclass axiom which to a class which does not allow any.

Reply:
No - the way confirmation works there should be no way in the world that they appear in the confirmed file. How? If it's obsolete it cannot have a subclass axiom. How can it then be confirmed?

Reply:
@matentzn OK, so I looked and this is happening because my `mondo.db` is from Nov 6, before this class got obsoleted.

I need to fix the NCIT robot error I [described in the OP](https://github.com/monarch-initiative/mondo-ingest/pull/717#issue-2713826952). Then, I can run this build properly, refreshing the prereqs, and this issue should go away.

Reply:
Alright! Error gone, re-ran, and now we are good!

This case no longer appears in that file. I double checked `sync-subClassOf.confirmed.tsv` and there are 0 obsolete entries now.

As for the error, I've seen this before with NCIT. It commonly had gotten memory issues before I upgraded my laptop, but also sometimes it gets `UnparsableOntologyException`. All I had to do was re-run and the error just didn't appear this time.

Reply:
@joeflack4 glad you sorted out the issue with previously having an obsolete Mondo class in the confirmed file! Was the `mondo.db` outdated (from Nov 6) when these files were generated because the files are from a mini build?

Reply:
Nov 6 just appears to be the last time that it was naturally refreshed in my development environment.

The mini build would have refreshed this, but there was a transient NCIT error that sometimes happens. So I temporary commented out some makefile prerequisites declarations and just ran it with the state of the files I had at the time, so that's why it was wonky.

The next time I ran things, the NCIT error did not re-occur; a mystery why this happens sometimes.

Reply:
@joeflack4 can you document this behavior with NCIT in a ticket or point me to an existing ticket if you've already created one? Also, if you run into a situation where you're stuck during a build run and need to comment out prerequisites, please include that information in the PR comment.